docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,123 @@
+# **<i class="fa fa-ticket"></i> PoS General FAQ**
+
+---
+
+#### **1. What is Decred's Proof-of-Stake system?**
+
+Besides Proof-of-Work (PoW) mining, Decred has a Proof-of-Stake (PoS) system. In this PoS system the participants buy tickets using their Decred coins (DCR). These tickets go into the PoS system ticket pool and can be called upon to vote on the previous block, like a lottery. There is a very small chance your ticket will not be called upon to vote.
+
+Every block 5 tickets are chosen from the PoS ticket pool. If at least 3 of those validate the previously mined block, the newly mined block is added to the chain, and both PoW miners and PoS stakers are paid. If the block is not validated by the PoS stakers the PoW miners do not get paid, but the PoS stakers do. This is to incentivize PoW miners to mine according to the wishes of the PoS stakers.
```

Comment:
this still reads weirdly:

> If at least 3 of those validate the previously mined block, the newly mined block is added to the chain, 

While this is true that the block is added, the block is always added no matter which way the vote goes.  What the vote changes is whether the transactions in the previous block are logically removed from the chain (as opposed to "physical" removal where the block doesn't actually exist in the chain anywhere).